frontend/CHANGELOG.md

```
@@ -1,5 +1,17 @@
 # Changelog
 
+## 0.9.0 (2025-11-18)
+
+## What's Changed
+* fix: update integration identifiers for Google services to use lowerc… by @Dhruv-Maradiya in https://github.com/theexperiencecompany/gaia/pull/317
```

markdownlint-cli2 (0.18.1)

Bare URL used

(MD034, no-bare-urls)